app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        copy, old_file, new_file = command.split()
+
+        if old_file != new_file or copy == "copy":
```

Comment:
Command has to start with `cp`
```suggestion
        if old_file != new_file or copy == "cp":
```